bindings/cs/MongoDB.Libmongocrypt/Status.cs

```diff
@@ -32,12 +32,30 @@ public Status()
             _handle = Library.mongocrypt_status_new();
         }
 
+        public Status(StatusSafeHandle handle)
```

Comment:
I don't see this method usages in the repo. Is it involved in native library functioning?

Reply:
explained above